janis_core/translations/wdl.py

```diff
@@ -2043,23 +2039,29 @@ def prepare_move_statements_for_input(ti: ToolInput):
         if ti.localise_file and not ti.presents_as:
             newlocation = "."
         elif not ti.localise_file and ti.presents_as:
-            newlocation = f'"`dirname ~{{{ti.id()}}}`/{ti.presents_as}"'
+            newlocation = f"`dirname ~{{{ti.id()}}}`/{ti.presents_as}"
             base = newlocation
         else:
             newlocation = ti.presents_as
-            base = f'"{ti.presents_as}"'
+            base = ti.presents_as
 
-        commands.append(wdl.Task.Command(f"cp -f ~{{{ti.id()}}} {newlocation}"))
+        commands.append(wdl.Task.Command(f"cp -f '~{{{ti.id()}}}' '{newlocation}'"))
 
     if it.secondary_files():
+        sec_presents_as = ti.secondaries_present_as or {}
+
         for s in it.secondary_files():
             sectag = get_secondary_tag_from_original_tag(ti.id(), s)
+            if ti.localise_file and not ti.presents_as:
+                # move into the current directory
+                dest = "."
+            else:
+                sectag = get_secondary_tag_from_original_tag(ti.id(), s)
```

Comment:
this `else` could be redundant since `sectag` has been defined with the same definition before `if`